### PR TITLE
EA-2754: Fix issues with TimeSpans

### PR DESCRIPTION
**Pull request by ikattey**

### CODE REVIEW

File: entity-management-solr/src/main/java/eu/europeana/entitymanagement/solr/service/SolrService.java [hunk 1]
```diff
@@ -73,7 +73,7 @@ public void afterPropertiesSet() {
     payloadMapper.addMixIn(Agent.class, SolrEntitySuggesterMixins.AgentSuggesterMixin.class);
     payloadMapper.addMixIn(
         Organization.class, SolrEntitySuggesterMixins.OrganizationSuggesterMixin.class);
-    payloadMapper.addMixIn(Timespan.class, SolrEntitySuggesterMixins.TimespanSuggesterMixin.class);
+    payloadMapper.addMixIn(TimeSpan.class, SolrEntitySuggesterMixins.TimespanSuggesterMixin.class);
```
Comment (gsergiu) [inline entity-management-solr/src/main/java/eu/europeana/entitymanagement/solr/service/SolrService.java]: we might want to update the name of the Mixin as well

File: entity-management-web/src/test/resources/content/timespan.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {
-"type":"Timespan",
+"type":"TimeSpan",
 "timestamp":"2020-12-10T19:42:56.163Z",
 "text":["000-tal","1:a seklet","1:a århundradet","000-talet","I wiek","1. århundrede","1. století","1. gadsimts","1. stoljeće","I amžius","1 век","1. század","1. Jahrhundert","século I","segle I dC","I secolo","1st century","siglo I","1. sajand","1ος αιώνας","I Secolo D.C.","Primo secolo","I Secolo","Secolo I","1° secolo","I sec. d.C.","I secolo d.C.","1st-century","1e eeuw","Secolul I","Ier siècle","Ensimmäinen vuosisata","I век","1 век","1ú haois","1. storočie","1. stoletje","segle I"],
-"suggest_filters":["Timespan"],
+"suggest_filters":["TimeSpan"],
```
Comment (gsergiu) [inline entity-management-web/src/test/resources/content/timespan.json:5]: This change will require an update for the Entity API specs.

**Review by gsergiu — COMMENTED** [review]
the name of the related mixin could be updated, need to confirm the change for the suggest filter

File: entity-management-solr/src/main/java/eu/europeana/entitymanagement/solr/service/SolrService.java (hunk 1 above)
Comment (ikattey) [reply entity-management-solr/src/main/java/eu/europeana/entitymanagement/solr/service/SolrService.java]: Indeed. Good catch!

File: entity-management-web/src/test/resources/content/timespan.json (hunk 2 above)
Comment (ikattey) [reply entity-management-web/src/test/resources/content/timespan.json:5]: Is that something to be done as part of this PR? 🙂

Comment (gsergiu) [reply entity-management-web/src/test/resources/content/timespan.json:5]: I supose not, but I would still like to get the green light from Hugo

**Pull request merged**